src/components/Main/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import { NavLink } from 'react-router-dom';
+
+import styled from 'styled-components';
+
+const Container = styled.div`
```

Comment:
div보다는 header로 사용되면 좋겠네요~
```suggestion
const Container = styled.header`
```